Source/ObjectiveDropboxOfficial/Shared/Handwritten/Networking/DBTransportBaseClient.m

```diff
@@ -82,7 +84,7 @@ - (NSDictionary *)headersWithRouteInfo:(NSDictionary<NSString *, NSString *> *)r
       [headers setObject:pathRootStr forKey:@"Dropbox-Api-Path-Root"];
     }
 
-    if (routeAuth && [routeAuth isEqualToString:@"app"]) {
+    if ([routeAuths containsObject:@"app"]) {
```

Comment:
this may break the user api client. for example, when a route has both "app" and "user" auth, this will always try to add the app credential instead of user's access token.